Tutorials/2_hello_world_parameterized/2_hello_world_parameterized_chain.py

```diff
@@ -0,0 +1,149 @@
+# Databricks notebook source
+# DBTITLE 1,Install RAG Studio packages
+# MAGIC %pip install databricks-rag-studio "mlflow@git+https://github.com/mlflow/mlflow.git@databricks-rag-studio"
+
+# COMMAND ----------
+
+# Before logging this chain using the driver notebook, you need to comment out this line.
+dbutils.library.restartPython() 
+
+# COMMAND ----------
+
+import mlflow
+from operator import itemgetter
+from langchain_core.output_parsers import StrOutputParser
+from langchain.schema.runnable import RunnableLambda
+from langchain_community.chat_models import ChatDatabricks
+from langchain_core.prompts import PromptTemplate
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ## Enable MLflow Tracing
+# MAGIC
+# MAGIC Enabling MLflow Tracing is required to:
+# MAGIC - View the chain's trace visualization in this notebook
+# MAGIC - Capture the chain's trace in production via Inference Tables
+# MAGIC - Evaluate the chain via the Mosaic AI Evaluation Suite
+
+# COMMAND ----------
+
+mlflow.langchain.autolog()
```

Comment:
since no streaming, this is okay, but i think this should move to the driver notebook instead of the chain notebook. that way we don't have to call autolog inside model serving. 

not blocking for this PR tho